src/core/services/theming/theming.js

```diff
@@ -1003,6 +1000,38 @@ function generateAllThemes($injector, $mdTheming) {
       }
     });
   }
+
+  // Split a string representing a CSS file producing an array with a rule in each element of it.
+  function splitCss(themeCss) {
+    var result = [];
+    var currentRule = '';
+    var openedCurlies = 0;
+    var closedCurlies = 0;
+
+    for (var i = 0; i < themeCss.length; i++) {
+      var character = themeCss.charAt(i);
+      // Remove break lines
+      currentRule += character !== '\n' ? character : '';
```

Comment:
I'm not seeing where line breaks where removed in the original, so you probably shouldn't do this.